content/docs/0.7.x/continuous_delivery/delivery_assistant/index.md

```diff
@@ -0,0 +1,123 @@
+            "teststrategy": "performance"
+        },
+        "id": "5bf26759-4afa-4045-8ccf-81bc398c2fcd",
+        "shkeptncontext": "654c80b0-4a02-4d36-96f3-7447df1cdf41",
+        "source": "gatekeeper-service",
+        "specversion": "0.2",
+        "time": "2020-09-17T14:41:14.039Z",
+        "type": "sh.keptn.event.approval.triggered"
+    }
+    ```
+* In addition, by using the command [`keptn send-event approval.finished`](../../reference/cli/commands/keptn_send_event_approval.finished/) the CLI will print a list of open approvals and allows you to choose which one you want to approve/decline:
+
+    ```console
+    $ keptn send event approval.finished --project=ck-sockshop --stage=production --service=carts
+    Starting to send approval.finished event
+    
+     OPTION	VERSION	EVALUATION	
+     (1)	0.11.2	n/a		
+    Select the option to approve or decline: 
+    1
+    Do you want to (a)pprove or (d)ecline: 
+    a
+    ID of Keptn context: 654c80b0-4a02-4d36-96f3-7447df1cdf41
+    ```
+
+
+## Bridge
+ 
+* To confirm/decline an approval, open the *Environment* view, click on the stage that has a pending approval. When there are pending approvals for a service available, the service tile is expanded automatically. Additionally you can filter the services, to see only services with pending approvals, by clicking on the blue icon.
+
+* By clicking on the green checkmark, the deployment will be approved. You can also decline by clicking on the red abort button. 
+
+    {{< popup_image
+      link="./assets/bridge_environment_assisteddelivery.png"
+      caption="Keptn Bridge Assisted Delivery">}}
+
+## API
+
+* Note down the keptn-context (`shkeptncontext`) and `id` (maps to `triggeredid`) of the [sh.keptn.event.approval.triggered](https://github.com/keptn/spec/blob/0.1.5/cloudevents.md#approval-triggered) CloudEvent, e.g., 
+```
+  "shkeptncontext": "654c80b0-4a02-4d36-96f3-7447df1cdf41",
+  "id": "5bf26759-4afa-4045-8ccf-81bc398c2fcd",
+```
+
+* Specify a valid Keptn CloudEvent of type [sh.keptn.event.approval.finished](https://github.com/keptn/spec/blob/0.1.5/cloudevents.md#approval-finished) and store it as JSON file, e.g., `approval_finished.json`
+
+```json
+{
+  "shkeptncontext": "654c80b0-4a02-4d36-96f3-7447df1cdf41",
+  "source": "https://github.com/keptn/keptn/cli#approval.finished",
+  "specversion": "0.2",
+  "type": "sh.keptn.event.approval.finished",
+  "contenttype": "application/json",
+  "triggeredid": "5bf26759-4afa-4045-8ccf-81bc398c2fcd",
+  "data": {
+    "approval": {
+      "result": "pass",
+      "status": "succeeded"
+    },
+    "deploymentstrategy": "blue_green_service",
+    "eventContext": null,
+    "image": "docker.io/keptnexamples/carts",
+    "labels": null,
+    "project": "ck-sockshop",
+    "service": "carts",
+    "stage": "production",
+    "tag": "0.11.2",
+    "teststrategy": "performance"
+  }
+}
+```
+
+* Trigger the approval finished event via request on `/event`:
```

Comment:
I would not say "Trigger", but rather: "Send the approval finished event to the `/event` endpoint.